build.gradle

```diff
@@ -123,8 +123,8 @@ subprojects {
     }
 
     ext {
-        jacksonVersion = "2.9.9"
-        jacksonDatabindVersion = "2.9.9"
+        jacksonVersion = "2.9.10"
+        jacksonDatabindVersion = "2.9.10"
```

Comment:
same as https://github.com/treasure-data/digdag/pull/1253? the PR was merged into v0_10.